store.go

```diff
@@ -2782,8 +2841,8 @@ func (s *store) ContainerParentOwners(id string) ([]int, []int, error) {
 	return nil, nil, ErrLayerUnknown
 }
 
-func (s *store) Layers() ([]Layer, error) {
-	lstore, err := s.LayerStore()
+func (s *store) DeduplicateLayers() ([]Layer, error) {
```

Comment:
Why isn't this called AllLayers() or RWLayers?
This name is very confusing.

Reply:
I renamed it to `RWLayers` :+1: